source/optional_features/index.html.md.erb

```diff
@@ -11,3 +11,5 @@ You can customise your payments pages with GOV.UK Pay in the following ways:
 * [use Welsh](/optional_features/welsh_language/#use-welsh-on-your-payment-pages) on payment pages
 * [use your own payment failure pages](/optional_features/use_your_own_error_pages)
 * [prefill fields](/optional_features/prefill_user_details/) on the **Enter card details** page
+
```

Comment:
some of these customisations are available with payment links and some aren't. 
- custom branding can be done on payment links
- welsh is available on payment links (it's in the set up)
- failure pages is API
- prefill fields is API

I'd be inclined not to change this page. it should be clear from the details of each one if it's in the API call or not.

Reply:
ok will change back